### ISSUE TITLE
Missing function to determine file date

### ISSUE THREAD

**Issue by DarthGandalf** [kvirc/KVIrc#1247]
**Reported by ctrlaltca on 07 Jan 2012 17:28:09 UTC**
Similarly to the stat() function, we need to implement a $file.something that is able to determine file dates:

```
               time_t    st_atime;   /* time of last access */
               time_t    st_mtime;   /* time of last modification */
               time_t    st_ctime;   /* time of last status change */
```

---

Migrated from: https://svn.kvirc.de/kvirc/ticket/1247

**Comment by DarthGandalf** [comment]
**Modified by ctrlaltca on 07 Jan 2012 17:28:28**
Description:

``` diff
--- 
+++ 
@@ -1,6 +1,6 @@
 Similarly to the stat() function, we need to implement a $file.something that is able to determine file dates:
-
+{{{
                time_t    st_atime;   /* time of last access */
                time_t    st_mtime;   /* time of last modification */
                time_t    st_ctime;   /* time of last status change */
-
+}}}

```

**Comment by DarthGandalf** [comment]
**Modified by HelLViS69 on 07 Jan 2012 17:52:23**
Owner: pragmaware → HelLViS69
Status: new → assigned

**Comment by DarthGandalf** [comment]
**Comment by HelLViS69 on 09 Jan 2012 22:03:36**
Implemented in #6045

---

Resolution: none → fixed
Status: assigned → closed
